components/02-molecules/edit/FieldsContext.tsx

```diff
@@ -4,6 +4,7 @@ import { isAddress } from "viem";
 import _ from "lodash";
 import { ResolvedEnsData, TextRecords } from "@/lib/utils/ensData";
 import { DecodedAddr } from "@ensdomains/ensjs/dist/types/types";
+import validateBitcoinAddress from "bitcoin-address-validation";
```

Comment:
why do we need a library for bitcoin validation?

Reply:
There is many different patterns for bitcoin addresses(P2PKH, P2SH, P2WPKH, P2TR), we need a library to check if the address is following one of those patterns correctly

Reply:
nice, sounds good